### ISSUE TITLE
Fix /service endpoint for 52N SOS 4.x using POX

### ISSUE THREAD

**Issue by nuest** [52North/sos4R#40]
``` r
> mySOS <- SOS(url = "http://sensorweb.demo.52north.org/sensorwebtestbed/sos/pox")
[sos4R] Created SOS for URL http://sensorweb.demo.52north.org/sensorwebtestbed/sos/pox 
> mySOS <- SOS(url = "http://sensorweb.demo.52north.org/sensorwebtestbed/sos/kvp")
Error: Method Not Allowed
> mySOS <- SOS(url = "http://sensorweb.demo.52north.org/sensorwebtestbed/sos/kvp", binding = "KVP")
[sos4R] Created SOS for URL http://sensorweb.demo.52north.org/sensorwebtestbed/sos/kvp 
> mySOS <- SOS(url = "http://sensorweb.demo.52north.org/sensorwebtestbed/service", binding = "KVP")
[sos4R] Created SOS for URL http://sensorweb.demo.52north.org/sensorwebtestbed/service 
> mySOS <- SOS(url = "http://sensorweb.demo.52north.org/sensorwebtestbed/service", binding = "POX")
Error: Unsupported Media Type
```

**Comment by nuest** [comment]
So far `POX` only works with a `dcpFilter`, see https://github.com/52North/sos4R/issues/36#issuecomment-451426830

**Comment by nuest** [comment]
@EHJ-52n @CarstenHollmann Do you think this can be solved at all or is there always an ambiguity?

**Comment by EHJ-52n** [comment]
@nuest 
The "unsupported media type" issue is fixed in my branch in 23c6afdbe7e437f0f9d8a897316a87c159a8aac5.

**Comment by nuest** [comment]
All commands above work now as expected.
